spec/requests/v1/visitor_can_see_list_of_recipes_spec.rb

```diff
@@ -0,0 +1,45 @@
+RSpec.describe 'GET articles index' do 
+  describe 'Listing articles' do 
+    let!(:recipes) { 2.times { create(:recipe) } }
+    let(:headers) {{ HTTP_ACCEPT: "application/json" }
+    before do
+      get '/v1/recipes'
+      headers: headers
+    end
+
+    it 'API returns 2 recipes' do
+      expect(response_json.count).to eq 2
```

Comment:
The question we need to consider is what the response should look like. I always advocate wrapping the collection in a key that holds the resource name. I know that you have done it differently in other projects, but I would like us to discuss this. 
```
{ "recipes" [

  ]
}
```